Toronto Star.js

```diff
@@ -56,10 +55,10 @@ function detectWeb(doc, url) {
 function getSearchResults(doc, checkOnly) {
 	var items = {};
 	var found = false;
-	var rows = ZU.xpath(doc, '//a[span[contains(@class, "story__headline")]]');
-	for (var i = 0; i < rows.length; i++) {
-		var href = rows[i].href;
-		var title = ZU.trimInternal(rows[i].textContent);
+	var rows = doc.querySelectorAll('a[data-test-id="mediacard"]');
+	for (let row of rows) {
+		let href = row.href;
+		let title = text(row, '*[data-test-id="mediacard-headline"]');
```

Comment:
This can just be `text(row, 'h3')`!

Reply:
I was about to argue that it's unlikely that the headline is exclusively `h3` nor that `h3` are exclusively headlines. But from the   pages I tested this does seem to be the case, lol.

It also made me realize that the other selector missed some edge cases